cli/sawtooth_cli/admin.py

```diff
@@ -19,13 +19,15 @@
 
 from sawtooth_cli.admin_command.genesis import add_genesis_parser
 from sawtooth_cli.admin_command.genesis import do_genesis
+from sawtooth_cli.admin_command.keygen import add_keygen_parser
+from sawtooth_cli.admin_command.keygen import do_keygen
 
 
 def do_admin(args):
     if 'SAWTOOTH_HOME' in os.environ:
         data_dir = os.path.join(os.environ['SAWTOOTH_HOME'], 'data')
     else:
-        data_dir = '/var/lib/sawtooth'
+        data_dir = os.path.join(os.sep, 'var', 'lib', 'sawtooth')
```

Comment:
The assumption that the os separator matters here is incorrect.  "/var/lib/sawtooth" is only valid for Linux distributions, not Windows.